helm/akhq/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: Kafka GUI for Apache Kafka to manage topics, topics data, consumers group, schema registry, connect and more...
 name: akhq
-version: 0.1.2
+version: 0.1.3
```

Comment:
Helm chart are generated from master branch, this version already exist

Reply:
Ah, apologies. Should I create a new PR with the helm changes to master and the Dockerfile changes to dev?

Reply:
Yes it's better :) I need to provide a doc about this ;)